cmd/gen-go-sample/testdata/sample.want

```diff
@@ -18,14 +18,18 @@ package main
 import(
  "context"
  "fmt"
+ "log"
```

Comment:
suggest blank line between stdlib imports and and non

Reply:
Ah this fooled heuristic for checking if an import is stdlib. We look for '.' because nonstdlibs are URLs.